### PR TITLE
Fix/365 dashboard logo

### PR DESCRIPTION
**Pull request by circlecube**
## Summary

Addresses issue #365 

## Relevant technical choices

Use existing classes to style logo in this instance and add modifier class to style the "mini" logo (for final and beta logos). Also includes update to rendered story for this widget.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/sass/components/global/_googlesitekit-logo.scss [hunk 1]
```diff
@@ -63,3 +63,23 @@
...
+		margin-left: $grid-gap-phone / 4;
+		width: 78px;
+	}
+
+	.googlesitekit-svg-logo-sitekit-beta {
+		height: 17px;
+		margin-left: $grid-gap-phone / 2;
+		width: 102px;
+	}
+}
```
Comment (felixarntz) [inline assets/sass/components/global/_googlesitekit-logo.scss]: This new CSS should also follow BEM as much as possible like the rest of this file:
* The new class should be `googlesitekit-logo--mini`.
* It should be nested in the original `.googlesitekit-logo` block.
* The three inner selectors that override the default properties should match the original selectors in the `.googlesitekit-logo` block (i.e. include the other variant, the original selectors are always pairs of two).

File: stories/wp-dashboard.stories.js [hunk 2]
```diff
@@ -39,7 +39,11 @@ storiesOf( 'WordPress', module )
 			<div id="dashboard-widgets">
 				<div className="metabox-holder">
 					<div id="google_dashboard_widget" className="postbox">
-						<h2 className="hndle ui-sortable-handle"><span><SvgIcon id="logo-g" height="19" width="19" /><SvgIcon id="logo-sitekit" height="17" width="78" /></span>
+						<h2 className="hndle ui-sortable-handle">
+							<span><div className="googlesitekit-logo googlesitekit-logo-mini">
```
[the remark this answers is not in the document]
Comment (circlecube) [reply stories/wp-dashboard.stories.js]: Updated to a `span`

File: includes/Core/Admin/Dashboard.php [hunk 3]
```diff
@@ -116,7 +116,7 @@ private function add_widgets() {
 
 		wp_add_dashboard_widget(
 			'google_dashboard_widget',
-			$logo . $logo_text,
+			'<div class="googlesitekit-logo googlesitekit-logo-mini">' . $logo . $logo_text . '</div>',
```
[the remark this answers is not in the document]
Comment (circlecube) [reply includes/Core/Admin/Dashboard.php]: Using `span` good catch.

File: assets/sass/components/global/_googlesitekit-logo.scss (hunk 1 above)
Comment (circlecube) [reply assets/sass/components/global/_googlesitekit-logo.scss]: -- in use now and the original selectors are repeated.

**Pull request merged**